train.py

```diff
@@ -57,6 +57,7 @@ def main():
   parser.add_argument("--epoch-size", type=int, default=100000000, dest='epoch_size', help="Number of positions per epoch.")
   parser.add_argument("--validation-size", type=int, default=1000000, dest='validation_size', help="Number of positions per validation step.")
   parser.add_argument("--param-index", type=int, default=0, dest='param_index', help="Indexing for parameter scans.")
+  parser.add_argument("--skip_early_plies", type=int, default=0, dest='skip_early_plies', help="Skip n plies from the start.")
```

Comment:
naming nit: i'd suggest `--early-fen-skipping` for naming consistency with the other "skipping" flags (ie. `--no-smart-fen-skipping`, `--no-wld-fen-skipping`, `--random-fen-skipping`). otherwise, deferring to sopel and vondele for review